spec/mailers/user_mailer_spec.rb

```diff
@@ -154,11 +154,12 @@
     end
   end
   
-  describe "invite request" do
+  describe "invitation from a user request" do
```

Comment:
I just want to check -- this tests invitations sent directly by users to someone's email, while the other one tests invites sent by admins or the queue, correct?

If that's the case, rather than test for the phrase  "like to join us, please sign up at the following address," which appears in both versions of the email, we should probably test the part of the email that's different. (The version for invitations sent by users says "%{user} has invited you to join our beta!" while the other version says "You've been invited to join our beta!" I know apostrophes are a problem, and the login bit might be as well, but we can definitely test the "has invited you" and "been invited" parts of the sentences.)